modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -48,15 +48,14 @@ spec:
       memory: "512Mi"
     reservedMemory:
       - numaNode: 0
-  limits:
-    memory: "1124Mi"
+        limits:
+          memory: "1124Mi"
     systemReserved:
       memory: "512Mi"
     topologyManagerPolicy: "single-numa-node" <1>
-    topologyManagerScope: "pod" <2>
+    topologyManagerScope: "pod"
 ----
 <1> `topologyManagerPolicy` must be set to `single-numa-node`.
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text):

> Do not start a sentence with a lowercase term; rewrite the sentence.

```suggestion
<1> The `topologyManagerPolicy` field must be set to `single-numa-node`.
```
Alternatively, "Set the `topologyManagerPolicy` field to `single-numa-node`."